tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -604,7 +604,9 @@ private static void notifyObservers(Span span) {
 
         @Override
         public void tag(Span.Builder target, String key, String value) {
-            target.putMetadata(key, value);
+            if (key != null && value != null) {
```

Comment:
Can you also update javadoc on the TagAdapter to explain the null behavior?

Reply:
Updated.